potassium-nitrite/src/test/kotlin/org/dizitart/kno2/ImportExportTest.kt

```diff
@@ -38,7 +38,7 @@ class ImportExportTest : BaseExternalTest() {
     @Test
     fun testImportExport() {
         schemaFile = (System.getProperty("java.io.tmpdir") + File.separator
-            + "nitrite" + File.separator + "schema.json")
+            + "nitriteIET" + File.separator + "schema.json")
```

Comment:
Please make it consistent with nitrite `ExporterImporterTest`. Or may be change the directory name as kno2 instead of nitrite if you feel it is more natural.

Reply:
Fixed